nextflow_schema.json

```
@@ -627,28 +627,152 @@
+                    "default": 0.06,
+                    "description": "Retain HMM hits that have an e-value lower than this.",
+                    "help_text": "This converts any prediction score below this cut-off to '0'.By doing so only values above this value will be used in the final AMPcombi table. To change the prediction score cut-off for all other AMP prediction tools, use instead `--amp_cutoff`.\n\n> Modifies tool parameter(s):\n> - AMPCOMBI: `--hmm_evalue`",
+                    "fa_icon": "fas fa-sort-numeric-up"
+                },
+                "amp_ampcombi_parsetables_windowstopcodon": {
+                    "type": "integer",
+                    "default": 60,
+                    "description": "Assign the number of codons used to look for stop codons, upstream and downstream of the AMP hit.",
+                    "help_text": "This assigns the length of the window size required to look for stop codons downstream and upstream of the CDS hits. In the default case, it walks 10 codons downstream and upstream of the AMP hit and writes whether a stop codon was found.\n\n> Modifies tool parameter(s):\n> - AMPCOMBI: `--window_size_stop_codon`",
+                    "fa_icon": "fas fa-stop-circle"
+                },
+                "amp_ampcombi_parsetables_windowtransport": {
+                    "type": "integer",
+                    "default": 11,
+                    "description": "Assign the number of CDSs upstream and downstream of the AMP to look for a transport protein.",
+                    "help_text": "This assigns the length of the window size required to look for a 'transporter' e.g. ABC transporter downstream and upstream of the CDS hits. This is done on CDS classification level.\n\n> Modifies tool parameter(s):\n> - AMPCOMBI: `--window_size_transporter`",
+                    "fa_icon": "fas fa-car-side"
+                },
+                "amp_ampcombi_parsetables_removehitswostopcodons": {
+                    "type": "boolean",
+                    "description": "Remove hits that have no stop codon upstream and downstream of the AMP",
+                    "help_text": "Removes any hits/CDSs that don't have a stop codon found in the window downstream or upstream of the CDS assigned by '--amp_ampcombi_parsetables_windowstopcodon'. We recommend to turn it on if the results will be used downstream experimentally.\n\n> Modifies tool parameter(s):\n> - AMPCOMBI: `--remove_stop_codons`",
+                    "fa_icon": "fas fa-ban"
+                },
+                "amp_ampcombi_parsetables_ampir": {
+                    "type": "string",
+                    "default": ".ampir.tsv",
+                    "description": "Assigns the file extension used to identify AMPIR output.",
+                    "help_text": "Assigns the ending of the input files -file extension- to allow AMPcombi to identify the tool output from the list of input files.\n\n> Modifies tool parameter(s):\n> - AMPCOMBI: `--ampir_file`",
```

```suggestion
                    "help_text": "Assigns the file extension of the input files to allow AMPcombi to identify the tool output from the list of input files.\n\n> Modifies tool parameter(s):\n> - AMPCOMBI: `--ampir_file`",
```
Should the value here be a variable string or should the user select from a list of strings? (Also for the amplify/macrel/hmmsearch further down in this file)